tools/Cmdline.hs

```diff
@@ -75,6 +67,9 @@ cliOptions =
     , Option ['b']     ["bdcs"]
         (ReqArg (\bdcs opts -> opts { optBDCS = bdcs }) "BDCS")
         (printf "Path to the content store directory (%s)" $ optBDCS defaultOptions)
+    , Option ['s']     ["socket"]
+        (ReqArg (\sock opts -> opts { optSocketPath = sock }) "SOCKET")
+        (printf "Path to a UNIX socket")
     ]
```

Comment:
We also need a new one for --group that defaults to 'weldr', this is the group owner of the socket.